python_modules/dagster/dagster/config/post_process.py

```diff
@@ -69,7 +69,7 @@ def _recursively_resolve_defaults(
     elif context.config_type.kind == ConfigTypeKind.SCALAR_UNION:
         return _recurse_in_to_scalar_union(context, config_value)
     else:
-        check.failed("Unsupported type {name}".format(name=context.config_type.name))
+        check.failed("Unsupported type {name}".format(name=context.config_type.key))
```

Comment:
nit: can change these to old format calls to f-strings if you are changing them already